crates/copybook-sequence-ring/src/claude.md

```
@@ -1,11 +1,12 @@
 <!-- SPDX-License-Identifier: AGPL-3.0-or-later -->
 # claude.md
 
-## Scope
-- Library sources for `copybook-sequence-ring` (Other layer).
+## Scope
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)